app/models/chargeback_rate_detail.rb

```diff
@@ -2,7 +2,8 @@ class ChargebackRateDetail < ActiveRecord::Base
   belongs_to :chargeback_rate
   belongs_to :detail_measure, :class_name => "ChargebackRateDetailMeasure", :foreign_key => :chargeback_rate_detail_measure_id
 
-  validates_numericality_of :rate
+  validates :rate, :numericality => true
```

Comment:
As this is just rails functionality, I would tend towards not testing it. 
But I'm not sure how the general project opinion is, so I can of course do, if this is common practice